config/settings.yml

```diff
@@ -0,0 +1,1387 @@
+      - "/etc/default/ntp*"
+      - "/var/log/messages*"
+      - "/var/log/cron*"
+      - BUILD
+      - GUID
+      - VERSION
+    :ping_depot: true
+    :ping_depot_timeout: 20
+  :level: info
+  :level_rails: info
+  :level_aws: info
+  :level_aws_in_evm: error
+  :level_kube: info
+  :level_kube_in_evm: error
+  :level_api: info
+  :level_api_in_evm: error
+  :level_fog: info
+  :level_fog_in_evm: error
+  :level_rhevm: info
+  :level_rhevm_in_evm: error
+  :level_scvmm: info
+  :level_scvmm_in_evm: error
+  :level_vim: warn
+  :level_vim_in_evm: error
+:log_depot:
+  :password:
+  :uri:
+  :username:
+:management_system:
+  :power_operation_expiration: 10.minutes
+:performance:
+  :capture_threshold:
+    :ems_cluster: 50.minutes
+    :host: 50.minutes
+    :storage: 60.minutes
+    :vm: 50.minutes
+  :capture_threshold_with_alerts:
+    :host: 20.minutes
+    :vm: 20.minutes
+  :concurrent_requests:
+    :hourly: 1
+    :realtime: 20
+  :history:
+    :initial_capture_days: 0
+    :keep_daily_performances: 6.months
+    :keep_hourly_performances: 6.months
+    :keep_realtime_performances: 4.hours
+    :purge_window_size: 1000
+  :host_overhead:
+    :memory: 2.01.percent
+    :cpu: 0.15.percent
+:policy_events:
+  :history:
+    :keep_policy_events: 6.months
+    :purge_window_size: 1000
+:product:
+  :maindb: ExtManagementSystem
+:reporting:
+  :format_by_class:
+    :Fixnum:
+      :function:
+        :name: number_with_delimiter
+    :Float:
+      :function:
+        :name: number_with_delimiter
+  :format_by_column: {}
+  :history:
+    :keep_reports: 6.months
+    :purge_window_size: 100
+  :precision:
+    :default: 2
+  :precision_by_column:
+    :slope: 4
+  :queue_timeout: 1.hour
+:repository_scanning:
+  :defaultsmartproxy:
+:server:
+  :case_sensitive_name_search: false
+  :company: My Company
+  :custom_logo: false
+  :events:
+    :disk_usage_gt_percent: 80
+  :host:
+  :listening_port: '443'
+  :mks_classid: 338095E4-1806-4BA3-AB51-38A3179200E9
+  :mks_version: 2.1.0.0
+  :monitor_poll: 5.seconds
+  :name: EVM
+  :rails_server: thin
+  :remote_console_type: VMRC
+  :role: database_operations,event,reporting,scheduler,smartstate,ems_operations,ems_inventory,user_interface,web_services,automate
+  :server_dequeue_frequency: 5.seconds
+  :session_store: cache
+  :startup_timeout: 300
+  :timezone: UTC
+  :worker_monitor:
+    :kill_algorithm:
+      :name: :used_swap_percent_gt_value
+      :value: 80
+    :miq_server_time_threshold: 2.minutes
+    :nice_delta: 1
+    :poll: 2.seconds
+    :start_algorithm:
+      :name: :used_swap_percent_lt_value
+      :value: 60
+    :sync_interval: 30.minutes
+    :wait_for_started_timeout: 10.minutes
+  :zone: default
+:session:
+  :interval: 60
+  :memcache_server: 127.0.0.1:11211
+  :memcache_server_opts: "-l 127.0.0.1"
+  :show_login_info: true
+  :timeout: 3600
+:smartproxy_deploy:
+  :queue_timeout: 30.minutes
+:smtp:
+  :authentication: login
+  :domain: mydomain.com
```

Comment:
Agreed...I went for the one-to-one conversion of the old .tmpl.ymls to the new to avoid changes.

Let's add it to the "follow up" list.